src/testdir/test_functions.vim

```diff
@@ -1930,56 +1930,53 @@ func Test_readdirex()
 endfunc
 
 func Test_readdirex_sort()
-  CheckUnix
-  CheckNotBSD
```

Comment:
Sorry, I meant removing only `CheckNotBSD`. I think `CheckUnix` is still needed.

Reply:
Hm, I thought we do not need this anymore. Anyhow, I re-added it back